FlightStreamDeck.Logics/Actions/GenericGaugeAction.cs

```diff
@@ -16,10 +16,42 @@ public class GenericGaugeSettings
         public float MaxValue { get; set; }
         public string ToggleValue { get; set; }
         public string DisplayValue { get; set; }
+        public string SubDisplayValue { get; set; }
+        public string Type { get; set; }
+        public string ValuePrecision { get; set; }
+    }
+
+    public class CustomGaugeSettings: GenericGaugeSettings
+    {
+        public string HeaderBottom { get; set; }
+        public string DisplayValueBottom { get; set; }
+        public bool DisplayHorizontalValue { get; set; }
+        public string ChartSplitValue { get; set; }
+        public int ChartThicknessValue { get; set; }
+        public int ChartChevronSizeValue { get; set; }
+        public string AbsValText { get; set; }
+
+        internal bool EmptyPayload
+        {
+            get =>
+                string.IsNullOrEmpty(Header) &&
+                string.IsNullOrEmpty(HeaderBottom) &&
+                string.IsNullOrEmpty(ToggleValue) &&
+                string.IsNullOrEmpty(DisplayValue) &&
+                string.IsNullOrEmpty(SubDisplayValue) &&
+                string.IsNullOrEmpty(DisplayValueBottom) &&
+                string.IsNullOrEmpty(ChartSplitValue) &&
+                string.IsNullOrEmpty(AbsValText) &&
+                string.IsNullOrEmpty(ValuePrecision) &&
+                MinValue == 0 &&
+                MaxValue == 0 &&
+                ChartThicknessValue == 0 &&
+                ChartChevronSizeValue == 0 && !DisplayHorizontalValue;
+        }
     }
 
     [StreamDeckAction("tech.flighttracker.streamdeck.generic.gauge")]
-    public class GenericGaugeAction : StreamDeckAction<GenericGaugeSettings>
+    public class GenericGaugeAction : StreamDeckAction<CustomGaugeSettings>
```

Comment:
It's a bit nitpicking but can you help to change this back to GenericGaugeSettings? I would really want to keep the convention of GenericGaugeAction ~ GenericGaugeSettings to not confuse myself.

I think GenericGaugeSettings is not used anywhere now other than a base class for CustomGaugeSettings, so I would prefer merging CustomGaugeSettings's property into GenericGaugeSettings.

Reply:
On it.